### DISCUSSION TITLE
Help installing Kube-Prometheus on a raspberry pi cluster

### DISCUSSION THREAD

**Discussion by vitorafgomes** [prometheus-operator/kube-prometheus#1693] [Q&A]
Hi, I already installed Kube-Prometheus as nodePort and everything worked but I installed it as load balance now and it is not working.

As NodePort she presented me this way.

![image](https://user-images.githubusercontent.com/5174765/159291508-ab02c856-aed7-440c-a465-8adda71563e6.png)

With load balance

![image](https://user-images.githubusercontent.com/5174765/159291724-0d7384cd-725f-437c-8ece-27925d992a76.png)

![image](https://user-images.githubusercontent.com/5174765/159291905-9388ea56-364e-43b4-9dd7-823ebc273d0b.png)

![image](https://user-images.githubusercontent.com/5174765/159292960-8b1bfe76-cdbd-47f1-8ac6-5a7a35fcf425.png)

**Comment by bryanasdev000** [comment]
If you open a port-forward the graphs are ok again?

Try to open your browser dev tools and check console log for Grafana.
